scalyr_agent/copying_manager.py

```diff
@@ -1026,6 +1026,18 @@ def generate_status(self, warn_on_rate_limit=False):
             for entry in self.__log_matchers:
                 result.log_matchers.append(entry.generate_status())
 
+            if self.__config.enable_health_check:
+                result.health_check_result = "Good"
+                if (
+                    time.time()
+                    > self.__last_attempt_time
```

Comment:
What is `__last_attempt_time` initialized to before it actually has made an attempt?  `None`?  We will need to handle that.

Reply:
Hm, I'm actually wondering why even have this ``enable_health_check`` config option.

I assume it doesn't add that much overhead and we could simply remove the flag and have it enabled by default? Or other people feel strongly about having a flag?

Reply:
Only thing I can think of is having `status -v` return an error code might cause some issues if the user isnt prepared for it, but I can't actually think of a good example of that. I'll let @czerwingithub comment but I would be fine with removing that option.

Reply:
Yeah, I agree w/ Tomaz.  It doesn't make sense to not always have this on.  My apologies for not also thinking of that earlier.